lib/providers/http/services/aens.js

```diff
@@ -186,21 +186,22 @@ class AENS extends HttpService {
    * Revoke a name
    *
    * @param nameHash
-   * @param fee
+   * @param account
    * @param options
    * @returns {Promise<*>}
    */
-  async revoke(nameHash, fee = 1, options = {}) {
+  async revoke(nameHash, account, options = {}) {
+    const { fee = 1 } = options
```

Comment:
This can be moved to the function args:
`{ fee = 1 } = {}` instead of `options = {}`